platform-common/src/main/java/com/softicar/platform/common/io/stream/InputStreamCollection.java

```diff
@@ -0,0 +1,84 @@
+package com.softicar.platform.common.io.stream;
+
+import com.softicar.platform.common.core.exception.ExceptionsCollector;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.function.Supplier;
+
+/**
+ * Manages the safe opening and closing of a {@link Collection} of
+ * {@link InputStream} objects.
+ * <p>
+ * This class implements {@link AutoCloseable} and guarantees the proper closing
+ * of all successfully opened {@link InputStream}, as long as the {@link #close}
```

Comment:
Suggestion: `{@link InputStream}` -> `{@link InputStream} instances`
(you also did that below)

Reply:
ok